### ISSUE TITLE
timeout or memory leak warning

### ISSUE THREAD

**Issue by stsmall** [kern-lab/FILET#5]
Hi @andrewkern @dschride ,
I have been running into a Warning when running FILET training. Anything to be worried about?
thanks,
@stsmall
Warning::
training set size after balancing: 270003
Checking accuracy when distinguishing among all 3 classes
Using extraTreesClassifier
/anaconda3/lib/python3.7/site-packages/joblib/externals/loky/process_executor.py:691: UserWarning: A worker stopped while some jobs were given to the executor. This can be caused by a too short worker timeout or by a memory leak.
  "timeout or by a memory leak.", UserWarning
GridSearchCV took 10745.41 seconds for 24 candidate parameter settings.
Results for extraTreesClassifier

**Comment by dschride** [comment]
Looks like this shouldn't affect your results: https://github.com/scikit-learn/scikit-learn/issues/14626
